requests/sessions.py

```diff
@@ -735,7 +735,12 @@ def session():
     """
     Returns a :class:`Session` for context-management.
 
+    .. deprecated:: 1.0.0
+
+        This method has been deprecated since version 1.0.0 and is only kept for
+        backwards compatibility. New code should use :class:`~requests.session.Session`
```

Comment:
I think this might need to be`requests.sessions.Session`.

Reply:
Whatever the module path is, that's what it should be. I was writing from memory and didn't intend for it to be copied verbatim.

Reply:
Updated.